components/inputs/input-checkbox.js

```diff
@@ -0,0 +1,146 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { checkboxStyles } from './input-checkbox-styles.js';
+import { classMap} from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+class InputCheckbox extends LitElement {
+
+	static get properties() {
+		return {
+			ariaLabel: { type: String, attribute: 'aria-label' },
+			checked: { type: Boolean },
+			disabled: { type: Boolean },
+			indeterminate: { type: Boolean },
+			name: { type: String },
+			notTabbable: { type: Boolean, attribute: 'not-tabbable' },
+			value: { type: String }
```

Comment:
In the old Polymer code, pretty much everything was reflecting back to the attribute. I removed that... which in theory could break something, but I feel it's doubtful and kind of silly to always-be-reflectin'. Our buttons suffered from the same thing unfortunately.